source/NVDAHelper.py

```diff
@@ -32,19 +33,20 @@
 
 
 versionedLibPath = os.path.join(globalVars.appDir, 'lib')
-if os.environ.get('PROCESSOR_ARCHITEW6432') == 'ARM64':
-	versionedLib64Path = os.path.join(globalVars.appDir, 'libArm64')
-else:
-	versionedLib64Path = os.path.join(globalVars.appDir, 'lib64')
+versionedLibARM64Path = os.path.join(globalVars.appDir, 'libArm64')
+versionedLibAMD64Path = os.path.join(globalVars.appDir, 'lib64')
 
 
 if not NVDAState.isRunningAsSource():
 	# When running as a py2exe build, libraries are in a version-specific directory
 	versionedLibPath=os.path.join(versionedLibPath,versionInfo.version)
-	versionedLib64Path=os.path.join(versionedLib64Path,versionInfo.version)
+	versionedLibAMD64Path = os.path.join(versionedLibAMD64Path, versionInfo.version)
+	versionedLibARM64Path = os.path.join(versionedLibARM64Path, versionInfo.version)
+
 
 _remoteLib=None
-_remoteLoader64=None
+_remoteLoaderAMD64 = None
+_remoteLoaderARM64 = None

```

Comment:
Could these get type hints?

```suggestion
_remoteLoaderAMD64: Optional[RemoteLoader] = None
_remoteLoaderARM64: Optional[RemoteLoader] = None
```

Reply:
I've added type hints now. But they had to be strings as they appear before the class definition.